src/ontology/uberon-edit.obo

```diff
@@ -184,7 +184,6 @@ property_value: doap-GitRepository https://github.com/cmungall/uberon/ xsd:anyUR
 treat-xrefs-as-has-subclass: EHDAA
```

Comment:
Doesn't look like all of the axioms that ended up i the header are accounted for by new content in OBO.  Usually there's some simple mistake causing divergence from OBO.  Occasionally we have logical axioms that need to be in an extension OWL file but don't see that here.

Reply:
Are you sure, I think they are all accounted for.. mainly the alt-id ones. Can you give me an example of an axiom unaccounted for?

Reply:
Was expecting to see obsolete stanzas in OBO in the diff, but I guess these are already in place, 

e.g. 

![image](https://user-images.githubusercontent.com/112839/128018352-45be31e2-38c1-42a3-8611-24b2a4782dfd.png)

Reply:
I made a ticket here:
https://github.com/obophenotype/uberon/issues/2018

I don't think any of it is critical, but if you think so, let me know which ones are really important.